infra/tf/k8s_infra/nats.tf

```diff
@@ -47,6 +47,13 @@ resource "helm_release" "nats" {
 		promExporter = {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +17]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->remove this<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1019?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->